BREAKING_CHANGES.md

```diff
@@ -3,34 +3,78 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
 
-- [7.7.0 - 2020/05/13](#770---20200513)
-  - [Known Issues](#known-issues)
-  - [GA support](#ga-support)
-  - [New branching model](#new-branching-model)
-  - [Filebeat container inputs](#filebeat-container-inputs)
-  - [Metricbeat upgrade issue](#metricbeat-upgrade-issue)
-  - [Metricbeat split values for daemonset and deployment](#metricbeat-split-values-for-daemonset-and-deployment)
-- [6.8.9 - 2020/05/13](#689---20200513)
-- [7.6.2 - 2020/03/31](#762---20200331)
-  - [Kibana default resources](#kibana-default-resources)
-- [7.6.0 - 2020/02/11](#760---20200211)
-  - [Elasticsearch default resources](#elasticsearch-default-resources)
-- [7.5.0 - 2019/12/02](#750---20191202)
-  - [Metricbeat kube-state-metrics upgrade](#metricbeat-kube-state-metrics-upgrade)
-- [7.0.0-alpha1 - 2019/04/17](#700-alpha1---20190417)
-  - [Elasticsearch upgrade from 6.x](#elasticsearch-upgrade-from-6x)
+- [7.8.0 - 2020/06/18](#780---20200618)
+    - [Stable Elasticsearch deprecated](#stable-elasticsearch-deprecated)
+    - [APM Server memory limit](#apm-server-memory-limit)
+    - [Elasticsearch service selector change](#elasticsearch-service-selector-change)
+  - [7.7.0 - 2020/05/13](#770---20200513)
+    - [Known Issues](#known-issues)
+    - [GA support](#ga-support)
+    - [New branching model](#new-branching-model)
+    - [Filebeat container inputs](#filebeat-container-inputs)
+    - [Metricbeat upgrade issue](#metricbeat-upgrade-issue)
+    - [Metricbeat split values for daemonset and deployment](#metricbeat-split-values-for-daemonset-and-deployment)
+  - [6.8.9 - 2020/05/13](#689---20200513)
+  - [7.6.2 - 2020/03/31](#762---20200331)
+    - [Kibana default resources](#kibana-default-resources)
+  - [7.6.0 - 2020/02/11](#760---20200211)
+    - [Elasticsearch default resources](#elasticsearch-default-resources)
+  - [7.5.0 - 2019/12/02](#750---20191202)
+    - [Metricbeat kube-state-metrics upgrade](#metricbeat-kube-state-metrics-upgrade)
+  - [7.0.0-alpha1 - 2019/04/17](#700-alpha1---20190417)
+    - [Elasticsearch upgrade from 6.x](#elasticsearch-upgrade-from-6x)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- Use this to update TOC: -->
 <!-- docker run --rm -it -v $(pwd):/usr/src jorgeandrada/doctoc --github -->
 
 
+# 7.8.0 - 2020/06/18
+
+### Stable Elasticsearch deprecated
+
+[Stable Elasticsearch chart][] is now deprecated in favor of
+[Elastic Elasticsearch chart][] (see [Stable Elasticsearch chart notice][]).
+
+Existing users of [Stable Elasticsearch chart][] can use the [migration guide][].
+
+### APM Server memory limit
+
+APM Servers default memory limit is increased in [#664][].
+
+This change may impact memory available resources capacity in your Kubernetes
+cluster.
+
+To come back to former default values, use the following values:
+
+```yaml
+resources:
+  limits:
+    memory: "200Mo"
```

Comment:
I think this should be `200Mi` ?

Reply:
good catch: fixed in https://github.com/elastic/helm-charts/pull/674/commits/00600239e7a51d68bf8db57cd68b77b734eb1e79